samza-core/src/main/scala/org/apache/samza/coordinator/stream/CoordinatorStreamWriterCommandLine.scala

```diff
@@ -68,4 +72,27 @@ class CoordinatorStreamWriterCommandLine extends CommandLine {
 
     value
   }
+
+  def genJobConfigs(config: MapConfig): MapConfig = {
```

Comment:
Rather than duplicating this code at two places, could you please extract them into a util method and move the duplicated code there.  Invoke the util method at all the places where you're calling this method.

Reply:
Yea was thinking to do that, will do!